### DISCUSSION TITLE
Trigger Custom Voice Commands

### DISCUSSION THREAD

**Discussion by viskor** [gopro/labs#328] [Feature Requests]
Hi.

Its possible to add another GoPro Voice Commands ?

When i have a probleme at road i tell : "CONNARD" and i want the gopro launch automaticaly record on this command.

https://youtube.com/shorts/JIRll01Kq34?feature=share

**Comment by dnewman-gpsw** [comment]
Unfortunately adding voice commands is very difficult, as the whole library of commands would need to be retrained in every language.  We have considered it, that is why I know it is hard to add a library of expletives.
